ziprofs.py

```diff
@@ -198,37 +167,43 @@ def readdir(self, path, fh):
         result.extend(subdirs)
         return result
 
-
     def release(self, path, fh):
-        zip_path = self.get_zip_path(path)
-        if not zip_path:
-            return os.close(fh)
-        return 0
+        if self.get_zip_path(path):
+            f = self._zip_fh[fh]
+            del self._zip_fh[fh]
+            return f.close()
+        else:
+            return os.close(fh >> 1)
 
     def statfs(self, path):
         stv = os.statvfs(path)
-        return dict((key, getattr(stv, key)) for key in ('f_bavail', 'f_bfree',
-            'f_blocks', 'f_bsize', 'f_favail', 'f_ffree', 'f_files', 'f_flag',
-            'f_frsize', 'f_namemax'))
+        return dict((key, getattr(stv, key)) for key in (
+            'f_bavail', 'f_bfree', 'f_blocks', 'f_bsize', 'f_favail',
+            'f_ffree', 'f_files', 'f_flag', 'f_frsize', 'f_namemax'
+        ))
 
 
 if __name__ == '__main__':
     parser = argparse.ArgumentParser(
         description='ZipROFS read only transparent zip filesystem.',
         formatter_class=argparse.ArgumentDefaultsHelpFormatter)
-    parser.add_argument('ROOT', nargs='?', help="filesystem root")
-    parser.add_argument('MOUNTPOINT', nargs='?', help="filesystem mount point")
+    parser.add_argument('root', nargs='?', help="filesystem root")
+    parser.add_argument('mountpoint', nargs='?', help="filesystem mount point")
```

Comment:
You've changed the argument names but you forgot to update the readme with the changed names